plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -283,6 +283,7 @@ Table update(TableInfo table)
     public void createSchema(DatasetInfo datasetInfo)
     {
         bigQuery.create(datasetInfo);
+        remoteDatasetIdCache.invalidate(datasetInfo.getDatasetId().getProject());
```

Comment:
@wendigo @hashhar @nineinchnick I pushed another small commit. Could you please take another look?